packages/SwingSet/docs/delivery.md

```diff
@@ -621,8 +638,9 @@ immediately after translation).
 The subject of a `syscall.resolve()` must either be a new Promise ID, or a
 pre-existing one for which the calling Vat is the Decider. The
 `KernelPromise` must be in the `Unresolved` state. If any of these conditions
-are not met, the Vat calling `resolve` will be terminated. It doesn't matter
-whether the Promise was allocated by this Vat or a different one.
+are not met, the Vat calling `resolve` will be terminated. Furthermore, the
+promise can only have been allocated by another vat unless the resolving vat is
```

Comment:
I can't parse that sentence. s/unless the/if the/ ?

Maybe "Furthermore, only pipelining-enabled vats are allowed to resolve promise IDs that they did not allocate." ?

Reply:
Done.